README.md

```diff
@@ -26,27 +26,37 @@ You can install it by following the [documentation](https://docs.oscar.grycap.ne
 
 ### 🧑‍💻 Setting Up the Configuration File
 
-The test suite uses environment variables to store sensitive information such as  endpoints and credentials.
+The test suite uses environment variables to store sensitive information such as endpoints and credentials. I'd recommend that you have two environment files. The first includes the cluster information, and the second contains the authentication process credentials. This way, you can switch between authentication processes such as EGI-CheckIn or Keycloak. Also, you can create one environment file that contains all the information.
```

Comment:
If you recommend two environment files, include templates for the two files (now we have a single .env-template.yaml file). Include the attributes in each file with sample fake values.  Also, include an example on how to specify the two files to the robot command.